java/client/src/main/java/glide/managers/CommandManager.java

```diff
@@ -489,7 +489,7 @@ public static <ArgType> void populateCommandWithArgs(
      * @param arguments The arguments to add to the builder.
      * @param outputBuilder The builder to populate with arguments.
      */
-    private static void populateCommandWithArgs(
+    public static void populateCommandWithArgs(
```

Comment:
why?

Reply:
This was from https://github.com/aws/glide-for-redis/pull/1754/files#diff-1fb2aa8a194f88dc10ebbce07da24cecf5f08d3d8cac507101968db9d45c6b1d. Was this by mistake?

Reply:
I think I changed it to make buildArgs work in transaction, and that didnt have the latest changes transaction has now